### PR TITLE
Redesign Settings Dialog

### PR DESCRIPTION
**Pull request by terrymyc**
In response to #414, this PR redesigns the settings dialog following the material design principles.

image

**On a large screen:**
image

image

image

**On a small screen:**
image

image

image

### CODE REVIEW

**Comment by JTeijema** [comment]
Looks very good! Helps the UX by a big margin.

**Comment by terrymyc** [comment]
The idea of the font size setting comes from my Android smartphone 👀

Screenshot_20210628-003006

**Comment by JTeijema** [comment]
> The idea of the font size setting comes from my Android smartphone 👀
No need to reinvent the wheel right, hahaha.

File: docs/source/features/settings.rst [hunk 1]
```diff
@@ -1,70 +1,50 @@
 Settings
 ========
 
+Click on Settings in the hamburger menu in the upper left cornor of ASReview LAB to configure your preferences.
```
Comment (J535D165) [inline docs/source/features/settings.rst:4]:
```suggestion
ASReview LAB offers the option to customize the screening appearance and functionality. 

1. Open ASReview LAB.
2. Click on Menu (top left).
3. Click on Settings.
```

File: docs/source/features/settings.rst [hunk 2]
```diff
@@ -1,70 +1,50 @@
...
 
-It is possible to enable or disable the :ref:`undo option <export-project>` in the settings menu:
-
-1. Open ASReview LAB.
-2. Click on the settings icon (top right).
-3. Uncheck the **Undo** field.
+Keyboard shortcuts
+~~~~~~~~~~~~~~~~~~
 
+You can press a key (or a combination of keys) to label a record as relevant or irrelevant, or to return to the previous decision during screening.
```
Comment (J535D165) [inline docs/source/features/settings.rst:39]:
```suggestion
You can press a key (or a combination of keys) to label a record as relevant 
or irrelevant, or to return to the previous decision during screening.
```

**Review by J535D165 — COMMENTED** [review]
Very nice improvement!

**Pull request merged**